src/lib/settings-context.tsx

```
@@ -35,6 +38,13 @@ export function SettingsProvider({ children }: { children: React.ReactNode }) {
     if (savedSettings) {
       try {
         const parsed = JSON.parse(savedSettings);
+
+        // Migration: Convert old betaCalculations to morningShading
+        if (parsed.betaCalculations !== undefined) {
+          parsed.morningShading = parsed.betaCalculations;
+          delete parsed.betaCalculations;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)